package.json

```diff
@@ -12,6 +12,9 @@
     "copy-webpack-plugin": "^11.0.0",
     "css-loader": "^6.7.3",
     "css-minimizer-webpack-plugin": "^4.2.2",
+    "datatables.net": "^1.13.2",
+    "expose-loader": "^4.0.0",
+    "jquery": "^3.6.3",
```

Comment:
same question.

Reply:
I moved the jquery and the datatables to the  dependency .